pkg/scheduler/metrics/queue.go

```diff
@@ -253,4 +273,7 @@ func DeleteQueueMetrics(queueName string) {
 	queueDeservedScalarResource.DeletePartialMatch(partialLabelMap)
 	queueCapacityScalarResource.DeletePartialMatch(partialLabelMap)
 	queueRealCapacityScalarResource.DeletePartialMatch(partialLabelMap)
+	knownScalarResourcesLock.Lock()
+	delete(knownScalarResources, queueName)
```

Comment:
You only delete the queue's key in the `knownScalarResources` map, but didn't delete the metrics